src/main/java/xyz/kyngs/herbot/util/ExceptionUtil.java

```diff
@@ -0,0 +1,14 @@
+package xyz.kyngs.herbot.util;
+
+public class ExceptionUtil {
+
+    public static boolean completedExceptionally(Runnable action) {
```

Comment:
finishedWithException is probably better name.

Reply:
Well I would rather stick with this name, because for example CompletableFuture uses isCompletedExceptionally.